manuscript-en/08-abstraction.md

```diff
@@ -255,11 +255,11 @@ async function submitLoginForm(event) {
 
 It's pretty compact, only 13 lines, but we can count three tasks in it:
 
-- Form serialization;
+- Data preparation;
 - Data validation;
 - Network requests.
 
-We can also count them by changing the conditions of a task and see _which code will change because of it_. For example, if we add a checkbox to the form, the serialization code will change:
+We can also count them by changing the conditions of a task and see _which code will change because of it_. For example, if we add a checkbox to the form, the data preparation code will change:
```

Comment:
```suggestion
We can also count them by changing the conditions of a task and see _which code will change because of it_. For example, if we add a checkbox to the form, the data extraction code will change:
```